interface-elements-for-web/articles/tree-list/layout-and-navigation/selection-and-navigation.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Selection and Navigation
+author: Alexey Kazakov
+legacyId: 4533
+---
+# Selection and Navigation
+## Navigation
+Use the Tree List's pager to switch between pages.
+
+![ASPxTreeList_Paging](../../../images/img7367.png)
+
+To change the number of nodes displayed within a Tree List, click the **Page Size** menu and select the needed number of nodes per page.
+
+## Selection
+
+Select and clear a check box to select and cancel the selection of the corresponding node. 
```

Comment:
check box -> check bo

Reply:
"select and cancel the selection" select and unselect?

Reply:
I used Microsoft style guide for choosing terminology. But the 'select and unselect' is better, I'll change. However we usually write check box.